hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineFactory.java

```diff
@@ -61,4 +61,13 @@ public Pipeline create(ReplicationType type, ReplicationFactor factor,
       List<DatanodeDetails> nodes) {
     return providers.get(type).create(factor, nodes);
   }
+
+  @VisibleForTesting
+  public PipelineProvider getProvider(ReplicationType type) {
```

Comment:
We don't need to expose this api for now. In the test where it is used, we can call pipelineManager.createPipeline instead.

Reply:
Done